react/package.json

```diff
@@ -21,5 +21,8 @@
     "apollo-cache-inmemory": "^1.6.5",
     "graphql": "^14.6.0",
     "typescript": "^3.9.6"
+  },
+  "resolutions": {
+    "@types/jest": "26.0.15"
```

Comment:
Should we use the same version at the devDependencies, too?

Line 16 of this same file:
https://github.com/vtex-apps/react-app-template/pull/70/files#diff-5a3f058f37c9436d08fa97bd30209f67c9a1a61d041b0ce27e48613dfb3a1a10R16?

Reply:
We could probably remove that line, actually. @types/jest is a dependency of @vtex/test-tools

Reply:
@danzanzini Done